src/main/scala/io/iohk/ethereum/db/storage/BlockBodiesStorage.scala

```diff
@@ -23,6 +24,12 @@ class BlockBodiesStorage(val dataSource: DataSource) extends KeyValueStorage[Blo
   override def valueDeserializer: (IndexedSeq[Byte]) => BlockBody =
     (encodedBlockBody: IndexedSeq[Byte]) => rlpDecode[BlockBody](encodedBlockBody.toArray)
 
+  override def get(key: BlockBodyHash): Option[BlockBody] = if (key == Config.Blockchain.genesisBlockHeader.hash) {
```

Comment:
Minor detail that I just noticed: the `BlockBodyHash` is the type used for the hash of the block right? It seems a little confusing as it seems to show that it is some sort of hash of the block body.